docs/pages/serving_a_federation.mdx

```diff
@@ -0,0 +1,209 @@
+
+### Set Registry URL (required)
+
+Pelican director needs to talk to the registry to get the public keys for namespaces. You will need to set `Federation.RegistryUrl` to the public endpoint of your registry
+for Pelican director to work.
+
+### Director as a Federation (optional)
+
+For Pelican origins and caches, they need to know where the _federation_ is so that they can
+attemp to join it. They know it either by manually passed configuration values or by requesting
+`Federation.DiscoveryUrl` endpoint, which contains various URLs to a federation's registry and director.
+You may supply your own endpoint to provide such information, or Pelican director can do it for you.
+
+This can be done by setting `Federation.DiscoveryUrl` of your _origins_ and _caches_ to the public url of your director, i.e. `https://<your-director-hostname>:<director-web-port>`. Make sure your director is up and running before starting your origins and caches.
+
+### Finish Setup
+
+That's it! Now you can serve the director by running the following command:
+
+```bash
+pelican director serve
+```
+
+and you will see the following message:
+
+```console
+Pelican admin interface is not initialized
+To initialize, login at https://<hostname>:8444/view/initialization/code/ with the following code:
+865309
+```
+
+By default, redirect runs on port `8444`. You may change the port number by passing `-p <port>` when serving
+the director, or by setting `Server.WebPort` in the configuration file.
+
+To finish setting up the web UI for the director, please refer to the previous section in the registry. Once finished the initialization and login with your admin password, you should be able to see the following page as an admin user (lists are expected to be empty at a fresh start):
+
+<ExportedImage src={"/pelican/director-homepage.png"} alt={"Image of director homepage in admin view"} />
+
+The "Origins" table and "Caches" table show the _active_ origins and caches in the federtaion.
+
+### Useful Configurations for Director
+
+There are a couple of configuration parameters you could use to customize the behavior of your registry. Here we highlight the ones that are most frequently set for an admin. You may refer to the full set of director parameters in the [Parameters page](https://docs.pelicanplatform.org/parameters#Director-DefaultResponse).
+
+#### `Director.DefaultResponse`
+
+Pelican director by default redirects object request from clients to the geospatially nearest cache in the federation, to improve data transfer efficiency. If a director is hosted at https://director.com, then a `GET` request to https://director.com/foo/bar.txt will be redirected to the nearest cache for namespace `/foo`. You may change the default response type of a redirect for director by setting `Director.DefaultResponse`. It can be either `cache` or `origin`. If set to `origin`, director will redirect the request to the origin, bypassing caches.
```

Comment:
```suggestion
A Pelican director by default redirects object request from clients to the geospatially nearest cache in the federation to improve data transfer efficiency. If a director is hosted at https://director.com, then a `GET` request to https://director.com/foo/bar.txt will be redirected to the nearest cache for namespace `/foo`. You may change the default response type of a redirect for director by setting `Director.DefaultResponse`. It can be either `cache` or `origin`. If set to `origin`, director will redirect the request to the origin, bypassing caches.
```